apps/ios/SimpleXChat/APITypes.swift

```diff
@@ -1194,8 +1194,136 @@ public struct UserProtoServers: Decodable {
     public var presetServers: [ServerCfg]
 }
 
+public enum OperatorTag: Decodable {
+    case simplex
+    case xyz
+    case demo
+}
+
+public struct ServerOperatorInfo: Decodable {
+    public var description: String
+    public var website: String
+    public var logo: String
+    public var largeLogo: String
+    public var logoDarkMode: String
+    public var largeLogoDarkMode: String
+}
+
+public let operatorsInfo: Dictionary<OperatorTag, ServerOperatorInfo> = [
+    .simplex: ServerOperatorInfo(
+        description: "SimpleX Chat preset servers",
+        website: "https://simplex.chat",
+        logo: "decentralized",
+        largeLogo: "logo",
+        logoDarkMode: "decentralized-light",
+        largeLogoDarkMode: "logo-light"
+    ),
+    .xyz: ServerOperatorInfo(
+        description: "XYZ servers",
+        website: "XYZ website",
+        logo: "shield",
+        largeLogo: "logo",
+        logoDarkMode: "shield",
+        largeLogoDarkMode: "logo-light"
+    ),
+    .demo: ServerOperatorInfo(
+        description: "Demo operator",
+        website: "Demo website",
+        logo: "decentralized",
+        largeLogo: "logo",
+        logoDarkMode: "decentralized-light",
+        largeLogoDarkMode: "logo-light"
+    )
+]
+
+public enum UsageConditionsAcceptance: Decodable, Hashable {
+    case accepted(date: Date)
+    case reviewAvailable(deadline: Date)
+    case reviewRequired
```

Comment:
it should be case required(deadline: Date?) - you don't have 3 cases here, it's 2.